crates/bootstrap_srv/src/store.rs

```diff
@@ -0,0 +1,191 @@
+//! This is a virtual-memory inspired tempfile store.
+//!
+//! ### Rationale
+//!
+//! - We don't need to persist anything in the store beyond a single
+//!   process invocation, because the infos are going to expire after
+//!   a matter of minutes anyways, and peers will continue to re-publish them.
+//! - We would like a server to be able to store more spaces and infos
+//!   than would reasonably fit in RAM.
```

Comment:
```suggestion
//!   than would reasonably fit in RAM.
//! - We want to avoid requiring on a database server as an additional system dependency for ease of deployment and reduced overhead.
```